php/arguments.php

```diff
@@ -0,0 +1,4 @@
+<?php
+// Read command line arguments and write to stdout
+echo strtolower($argv[1]) . PHP_EOL;
```

Comment:
I used `echo` in all the scripts because that is what ChatGPT wrote, but also could have used `print`. Not sure if it makes a difference